porch/api/porchconfig/v1alpha1/types.go

```diff
@@ -142,6 +144,16 @@ type FunctionRef struct {
 	Name string `json:"name"`
 }
 
+const (
+	// Type of the Repository condition.
+	RepositoryAvailable = "Available"
```

Comment:
Does this condition reflect the reconciliation status of the library (or somewhat equivalent to that?)

If so, I think we should try to adopt one of the two patterns supported by the [kstatus library](https://github.com/kubernetes-sigs/cli-utils/tree/master/pkg/kstatus#conditions). This means that our other tools will automatically be able to understand the conditions.

Reply:
Oh, nice. I was looking for some prior art and the k8s meta only mentioned "Available" as standard.

I would recommend we go with the `Ready` clause: https://github.com/kubernetes-sigs/cli-utils/tree/master/pkg/kstatus#the-ready-condition

which would be achieved simply by renaming using `Ready` instead of `Available`.